strong_resources.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "jsonapi_compliable", "~> 0.6"
+  spec.add_dependency "jsonapi_errorable", "~> 0.9.0"
```

Comment:
I'd remove this dependency and check if errorable is defined. I realize it's a somewhat pedantic point, doubt anybody is using this independent of suite, but I think it's probably a better long-term path to success. Imagine when this lib gets refactored to dry-types or some other kind of refactoring, we might not need this kind of integration.